server/plugin/api.go

```diff
@@ -136,8 +138,8 @@ func (p *Plugin) handleLink(w http.ResponseWriter, r *http.Request) {
 	project := serializers.ProjectDetails{
 		MattermostUserID: mattermostUserID,
 		ProjectID:        response.ID,
-		ProjectName:      response.Name,
-		OrganizationName: body.Organization,
+		ProjectName:      cases.Title(language.Und).String(body.Project),
```

Comment:
cases.Title(language.Und).String what does this do i am not familiar with this ?

Reply:
Converts the string to title case